spec/rails_helper.rb

```diff
@@ -74,8 +74,14 @@ def request_fixture(name)
   File.read(Rails.root.join('spec', 'fixtures', 'requests', name))
 end
 
+$attachments = {}
```

Comment:
I think it's worth leaving a comment explaining why this exists in the first place, otherwise 👍🏽 should help with IO overhead

Reply:
when I finalize everything, I'll clean it up and leave a comment